README.md

```diff
@@ -210,6 +210,7 @@ Hurray! You successfully made a contribution! 🎉
   - [Ankit Ojha](https://github.com/ankitojha07)
   - [Ankit Warbhe](https://github.com/ankitwarbhe/)
   - [annu](https://github.com/annu12340)
+  - [Anurag Kumar](https://github.com/kranurag7)
```

Comment:
Your name should be on like 220 after [this](https://github.com/EddieHubCommunity/hacktoberfest-practice/pull/746/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R220)